rpc/core/types/responses.go

```diff
@@ -35,6 +35,10 @@ type ResultCommit struct {
 	CanonicalCommit    bool `json:"canonical"`
 }
 
+type ResultDataAvailabilityHeader struct {
+	types.DataAvailabilityHeader `json:"data_availability_header"`
```

Comment:
Make this consistent throughout the code base.  We often use DAHeader in convos but this is also what block explorers will show etc. `data_availability_header` might be more expresive.